packages/shared-ux/error_boundary/src/services/error_service.ts

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
nit: this would be the most important file to add unit tests to, IMHO, as it will contain all the business logic of working with error instances.